### DISCUSSION TITLE
1 Proof but nothing happened. Why?. I am crying...Anyone can help me?

### DISCUSSION THREAD

**Discussion by nonogrub** [Chia-Network/chia-blockchain#6043] [Support]
![Captura de pantalla 2021-05-26 a las 8 09 45](https://user-images.githubusercontent.com/83576173/119611058-722da600-bdfa-11eb-92c2-93f7f51c68cc.png)

**Comment by lvcivs** [comment]
Finding a proof does not guarantee that you receive an award. Usually, you will, but not always. 

You can check your wallet address on chiaexplorer.com to see whether a reward was paid out.

**Reply by nonogrub** [reply]
Thanks for your reply. Is strange because if I copied an "receive address" from my GUI Wallet, chia explorer can't check nothing...xch1qek84zrqs5kdudwdajz9d8n2zf4d4m5srvhc6xpkg79tkzm0vw4qjga95k
But if I use chia show wallet from CLI the address received is valid in Chiaexplorer and show my mojos received form Faucet...

**Comment by BrandtH22** [comment]
@nonogrub , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
